### DISCUSSION TITLE
Any plans to support ECS system, data oriented scripting?

### DISCUSSION THREAD

**Discussion by MyelinsheathXD** [ezEngine/ezEngine#641] [Ideas]
When  Machine Learning agents needs to train in an simulated environment, they all the time need to run in multi threaded gameplay logic .

Any plans to support ECS system, which is data oriented scripting?

**Comment by jankrassnigg** [comment]
It already is a data oriented ECS, see
http://ezengine.net/pages/docs/runtime/world/world-overview.html

And with C++ you get the best speed possible.
